README.md

```diff
@@ -42,45 +61,183 @@ If we are happy with what we see we can detach the terminal with `Ctrl-p``Ctrl-q
 To reattach to the terminal (to see logging) run:
 
         $ docker attach mynodered
-        
-If you need to restart the container (e.g. after a reboot or restart of the Docker daemon)
+
+If you need to restart the container (e.g. after a reboot or restart of the Docker daemon):
 
         $ docker start mynodered
 
-and stop it again when required
+and stop it again when required:
 
         $ docker stop mynodered
 
 _**Note** : this Dockerfile is configured to store the flows.json file and any
 extra nodes you install "outside" of the container. We do this so that you may rebuild the underlying
 container without permanently losing all of your customisations._
 
-## Images
-
-The following images are built for each Node-RED release, using a Node.js v6 base image.
+## Image Variations
+The Node-RED images have different variations and are supported by manifest lists (auto-detect architecture).
 
-- **latest** - uses [official Node.JS v6 base image](https://hub.docker.com/_/node/).
-- **slim** uses [Alpine Linux base image](https://hub.docker.com/r/mhart/alpine-node/).
-- **rpi** uses [RPi-compatible base image](https://hub.docker.com/r/hypriot/rpi-node/).
+The tag naming convention is `<node-red-version>-<node-version>-<architecture>`, where:
+- `<node-red-version>` is the Node-RED version.
+- `<node-version>` is the Node JS version, either v6 or v8, based on Alpine or Debian slim images.
+- `<architecture>` is the architecture of the Docker host system, either amd64, arm32v6, arm32v7, arm64.
 
-Using Alpine Linux reduces the built image size (~100MB vs ~700MB) but removes
+The Node-RED images are based on the [official Node JS v8 and Node JS v6 images](https://hub.docker.com/_/node/), which are based on Alpine Linux or Debian Linux (slim).
+Using Alpine Linux or Debian Linux (slim) reduces the built image size (~100MB vs ~700MB), but removes
 standard dependencies that are required for native module compilation. If you
-want to add modules with native dependencies, use the standard image or extend
-the slim image with the missing packages.
+want to add dependencies with native dependencies, extend the Node-RED image with the missing packages on running containers or build new images.
+
+### Node.JS v8 based images
```

Comment:
Should match the versions being built

Reply:
Taking care of this in the new coming PR.